src/code/z_view.c

```diff
@@ -276,16 +290,18 @@ s32 func_800AAA9C(View* view) {
 
     OPEN_DISPS(gfxCtx, "../z_view.c", 596);
 
+    // Apply viewport
     vp = Graph_Alloc(gfxCtx, sizeof(Vp));
     LogUtils_CheckNullPointer("vp", vp, "../z_view.c", 601);
     View_ViewportToVp(vp, &view->viewport);
     view->vp = *vp;
 
-    func_800AA550(view);
+    View_ApplyScissor(view);
 
     gSPViewport(POLY_OPA_DISP++, vp);
     gSPViewport(POLY_XLU_DISP++, vp);
 
+    // Apply perspective projection
```

Comment:
I don't know if "Apply" is appropriate here. First, this allocates space on the heap to store the projection matrix. Then it saves the pointer to it in the appropriate variables. Then, `guPerspective` is where the perspective matrix is actually calculated. "Applying" makes me think the projection is already obtained, and is being "applied" to a different matrix to transform it into the appropriate coordinate systems.

Reply:
OK, have removed the word "Apply" here and nearby. 46ad1a2cb

Reply:
@engineer124 `Graph_Alloc` is essentially the same as `POLY_OPA_DISP++` except it appends (or, prepends, I guess) to the tail instead of the head. The memory area is the same for both, `GfxPool.polyOpaBuffer` (in `gGfxPools`)